gno.land/pkg/gnoland/genesis.go

```diff
@@ -58,6 +59,42 @@ func LoadGenesisBalancesFile(path string) ([]Balance, error) {
 	return balances, nil
 }
 
+// LoadGenesisParamsFile loads genesis params from the provided file path.
+func LoadGenesisParamsFile(path string) ([]Param, error) {
+	// each param is in the form: key.kind=value
+	content := osm.MustReadFile(path)
```

Comment:
Why the must?

`LoadGenesisParamsFile` already does error returns

Reply:
187d1e50e

It was copied from the two other existing LoadXXX files, which I also fixed.